packages/base/card-api.gts

```diff
@@ -3023,6 +3023,13 @@ export class Box<T> {
     }
   }
 
+  get containingBox(): Box<any> | undefined {
+    if (this.state.type === 'root') {
+      return undefined;
+    }
+    return this.state.containingBox;
+  }
```

Comment:
it was necessary to expose in our model API the ability to look upwards to find the card that contains the current model (which bottoms out at a field)

Reply:
I'm concerned about this API's potential for abuse. We really don't want fields reaching up to change the card they are contained in. Would an alternative approach be for the containing card to be provided to the LinksTo(Many)Editor, perhaps via dynamic context?

Reply:
That’s an interesting idea. Although would that expose us to the same level of abuse only via the @context?

Reply:
Also keep in mind that depending on how the field/card is composed the item that the field is contained in might be another field (I discovered this from our tests), so you need a way to traverse up the containment until you hit the actual card in order to determine the realm.

Reply:
Another idea here is that instead of trying to provide a capability to look up the containment, we pass the realm url of the containing card down (via the dynamic context) so that it’s available at the leaf fields (so a nested field of a field has the realm url of the containing card). What do you think of that idea?

Reply:
actually i think i see a better way, and probably this is what you were talking about the whole time: the dynamic context has the actual card in it--so there is no need to traverse the containment